app/routes/chatbot.js

```diff
@@ -229,14 +230,21 @@ router.post('/', (req, res) => {
       // We don't want to send an error back as response, but instead deliver success to Mobile
       // Commons and deliver the Campaign Closed message back to our User.
       const msg = controller.renderResponseMessage(scope, 'campaign_closed');
-      scope.user.postMobileCommonsProfileUpdate(mobileCommonsOIP, msg);
+      scope.user.postMobileCommonsProfileUpdate(scope.oip, msg);
 
       return res.send(gambitResponse(msg));
     })
     .catch(err => {
+      if (err.message === 'broadcast declined') {
+        const declinedMessage = 'K, no prob!';
```

Comment:
TODO: Minimal next step here is to add CampaignBot `msg_declined_broadcast` property to set our No Response broadcast.

Ideal fix: Build out a select campaign menu to list all current CampaignBot Campaigns, prompting User to text in a keyword to signup for one of those instead. Refs #713. Could potentially send to the Chatbot OIP instead of AgentView to handle responses with a "Sorry I didn't get that -- please text a keyword or {{command}} for Agent View"